insight-engine/latest/admin/monitor.md

```diff
@@ -177,6 +177,37 @@ Use the following setting to specify an option core for the report. If it is abs
 
 You can also fix index issues, check the index cache and backup individual indexes by using JMX. The status of the index can be checked using the JMX client on the **JMX MBeans > Alfresco > solrIndexes > `<store>`** tabs. The default view is the Solr core summary. The operations run the same consistency checks that are available by URL.
 
+### Troubleshooting - unsearchable content
+
+If content becomes unsearchable for some documents, it is possible that the necessary transformations failed and the documents are currently indexed with a `transform_failed` status.
+
+> **Note:** For Enterprise installations (assuming the reason for these failed transformations is the temporary unavailability of the Transform Engines (T-Engines)) it is recommended to verify that the T-Engines do get back up and running, and to check that ActiveMQ was and still is available. In that case, the documents indexed with a `transform_failed` status should be automatically reindexed once the T-Engines come back up and the prior ActiveMQ messages are consumed, without any further steps being required.
```

Comment:
Hi @damianujma - I've changed the wording here - is it still technically correct? 
```suggestion
> **Note:** For Enterprise installations (assuming the reason for these failed transformations is the temporary unavailability of the Transform Engines (T-Engines)), it is recommended to verify that the T-Engines start running again, and check that ActiveMQ is still available. In that case, the documents indexed with a `transform_failed` status should be automatically reindexed once the T-Engines start running again and the earlier ActiveMQ messages are consumed, without any further steps being required.
```

Reply:
LGTM.